main/FatesHistoryInterfaceMod.F90

```diff
@@ -1795,6 +1803,17 @@ subroutine update_history_dyn(this,nc,nsites,sites)
                              (ccohort%lmort_direct+ccohort%lmort_collateral+ccohort%lmort_infra) * ccohort%n
                        hio_m8_si_scls(io_si,scls) = hio_m8_si_scls(io_si,scls) + &
                              ccohort%frmort*ccohort%n
+			     
```

Comment:
@rgknox The calculation of discrimination is relevant to C3 plant at this point, grass should be included. Please fix this for us.   It may also an be error that it is affecting mortality calculation. it was our intention to create a  d13C related criterion to kill trees (as i presented in the NGEE-Tropics and AGU meetings). However, at this point, as we include this component to calculate discrimination only, we should not have any effect of on mortality. Thanks.